package.json

```diff
@@ -21,7 +21,7 @@
     "start": "npm run compile && electron build/app.js",
     "debug": "npm run compile && electron build/app.js --inspect-brk=9229",
     "build-all": "npm run compile && electron-builder --win --mac --linux --publish never",
-    "build-mac": "npm run compile && electron-builder --mac --publish never",
+    "build-mac": "npm run compile && electron-builder --mac --universal --publish never",
```

Comment:
The universals are 2x the size which we want to avoid if we can here. This added about 200 megs to Workstation the last time we tried it.

Reply:
Right yeah but I thought it sorts out couple of other different binaries issues.  So with this we would have 1 generic artifact build for x64 and arm64 and if we are using different arch binaries that will lead the conflicting artifact name which is : `chef-workstation-app-0.2.105-darwin` for the both arm64 and z64
Should we avoid it and go with different arch binaries ?

Reply:
Done with different arch binaries